rest/route/test.go

```diff
@@ -16,7 +16,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-// testGetHandler is the MethodHandler for the GET /tasks/{task_id}/tests route.
+///////////////////////////////////////////////////////////////////////////////
+//
+// GET  /tasks/{task_id}/tests
```

Comment:
nit: remove double spaces between `GET` and `/tasks`.

Reply:
Done.